blockchain/types/src/quorum_set.rs

```diff
@@ -0,0 +1,189 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+use alloc::{
+    string::{String, ToString},
+    vec,
+    vec::Vec,
+};
+use core::{
+    hash::{Hash, Hasher},
+    str::FromStr,
+};
+use mc_common::{NodeID, ResponderId};
+use mc_crypto_digestible::Digestible;
+use mc_crypto_keys::Ed25519Public;
+use prost::{Message, Oneof};
+use serde::{Deserialize, Serialize};
+
+/// The quorum set defining the trusted set of peers.
+#[derive(Clone, Deserialize, Digestible, Message, Ord, PartialOrd, Serialize)]
+pub struct QuorumSet {
+    /// Threshold (how many members do we need to reach quorum).
+    #[prost(uint32, required, tag = 1)]
+    pub threshold: u32,
+
+    /// Members.
+    #[prost(message, repeated, tag = 2)]
+    pub members: Vec<QuorumSetMemberWrapper>,
+}
```

Comment:
I think James held up this PR based on the QuorumSet duplication - should we schedule some time with James to what to do about it?

Reply:
I figured out how to reuse `mc_consensus_scp_core::QuorumSet`